Cargo.toml

```diff
@@ -58,6 +58,11 @@ x11rb = { version = "0.9.0", optional = true }
 xkbcommon = "0.4.0"
 scan_fmt = { version = "0.2.3", default-features = false }
 
+# Vulkan dependencies
+# TODO: Pending new release
+scopeguard = { version = "1.1.0", optional = true }
+ash = { git = "https://github.com/ash-rs/ash", optional = true }
```

Comment:
The ash release with lifetimes instead of builders will not come for some time.

For now I've chosen a revision that will be part of the planned `0.37.1` release of ash and made the changes to the codebase to reflect that.